makefiles/default-channel.inc.mk

```diff
@@ -11,3 +11,7 @@ else
     CFLAGS += -DIEEE802154_DEFAULT_CHANNEL=$(DEFAULT_CHANNEL)
   endif
 endif
+
+# Set a custom pan id if needed
+DEFAULT_PAN_ID ?= 0x23
+CFLAGS += -DIEEE802154_DEFAULT_PANID=$(DEFAULT_PAN_ID)
```

Comment:
This will cause `IEEE802154_DEFAULT_PANID` always to be added to `riotbuild.h`, disregarding if an IEEE 802.15.4 radio is present or not... Not sure this is desirable.

Reply:
You are right - and that file needed some cleanup anyway.
I rewrote my patch so that now the `CFLAGS` are only updated if the `make` variables are set.

I have my doubts that at86rf212b is the only sub-GHz radio, but I'd rather leave this can of worms sealed for now.

Reply:
> I have my doubts that at86rf212b is the only sub-GHz radio, but I'd rather leave this can of worms sealed for now.

You already did ;-)